src/planner/dart/ConfigurationToConfiguration.cpp

```diff
@@ -0,0 +1,75 @@
+#include "aikido/planner/dart/ConfigurationToConfiguration.hpp"
+
+namespace aikido {
+namespace planner {
+namespace dart {
+
+//==============================================================================
+ConfigurationToConfiguration::ConfigurationToConfiguration(
+    statespace::dart::ConstMetaSkeletonStateSpacePtr stateSpace,
+    ::dart::dynamics::ConstMetaSkeletonPtr metaSkeleton,
+    const statespace::dart::MetaSkeletonStateSpace::State* goalState,
+    constraint::ConstTestablePtr constraint)
+  : Problem(stateSpace, std::move(constraint))
+  , mMetaSkeletonStateSpace(stateSpace)
+  , mMetaSkeleton(std::move(metaSkeleton))
+  , mStartState(std::move(mMetaSkeletonStateSpace->createState()))
```

Comment:
@jslee02 Are these explicit `std::move`s for `mStartState` and `mGoalState` necessary? Based on my limited understanding of rvalues, those expressions might already be rvalues and therefore `std::move` is unnecessary?

Reply:
Yeah, I agree with this. @jslee02 can you confirm?